doc/user/abs_backup.md

```diff
@@ -0,0 +1,95 @@
+# Backups using Azure Blob Service (ABS)
+
+Etcd backup operator backups the data of an etcd cluster running on Kubernetes to a remote storage such as Azure Blob Service (ABS). If it is not deployed yet, please follow [here](walkthrough/backup-operator.md#deploy-etcd-backup-operator) to deploy it.
+
+## Setup Azure Secret
+
+Create a secret file `abs-secret.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: abs-credentials
+  namespace: etcd
+type: Opaque
+stringData:
+  storage-account: "<storage-account>"
+  storage-key: "<storage-key>"
+  cloud: "<cloud>"
+```
+
+In above secret, following data should be provided:
+
+- `storage-account`: the name of Azure storage account.
+- `storage-key`: the storage key of storage account.
+- `cloud`: the name of cloud environment, e.g. `AzurePublicCloud` or `AzureChinaCloud`.
+
+And then create the secret:
+
+```sh
+kubectl apply -f abs-secret.yaml
+```
+
+## Create EtcdBackup CRD
```

Comment:
It’s just a naming, and I’m just nitpicking, but is it CRD (definition) or just an instance of CRD? So maybe just resource instead of CRD?

Reply:
Agree. it should be CR instead of CRD.

Reply:
yep, it should be CR